connexion/middleware/swagger_ui.py

```diff
@@ -151,12 +151,12 @@ async def redirect(request):
             url += "/"
             return RedirectResponse(url=url)
 
-        self.router.add_route(methods=["GET"], path=console_ui_path, endpoint=redirect)
+        self.router.add_route(methods=["GET"], path=console_ui_path or "/", endpoint=redirect)
```

Comment:
I would rather change the way the path is stripped above.

```python
console_ui_path = self.options.swagger_ui_path.strip().rstrip("/") + "/"
```

and removing the leading "/" from the suffixes we add everywhere.